ghc-bignum-orphans (and potentially a few other packages) do not ship
a Setup.hs and instead rely on cabal to be installed which makes
boostraping very difficult. For these packages automatically create a
Setup.hs manually for now.

diff -upr b/bootstrap/bootstrap.py a/bootstrap/bootstrap.py
--- b/bootstrap/bootstrap.py	2021-11-17 19:31:03.748302012 +0100
+++ a/bootstrap/bootstrap.py	2021-11-17 19:31:37.522207586 +0100
@@ -17,6 +17,7 @@ from enum import Enum
 import hashlib
 import logging
 import json
+import os
 from pathlib import Path
 import platform
 import shutil
@@ -180,6 +181,11 @@ def install_dep(dep: BootstrapDep, ghc:
         shutil.unpack_archive(tarball.resolve(), UNPACKED, 'gztar')
         sdist_dir = UNPACKED / f'{dep.package}-{dep.version}'
 
+        setup_fp = sdist_dir / 'Setup.hs'
+        if not os.path.isfile(setup_fp):
+            with open(setup_fp, 'w') as setup_hs:
+                setup_hs.write('module Main where\nimport Distribution.Simple\nmain = defaultMain\n')
+
         # Update cabal file with revision
         if dep.revision is not None:
             shutil.copyfile(cabal_file, sdist_dir / f'{dep.package}.cabal')
